openfe/protocols/openmm_septop/femto_utils.py

```diff
@@ -0,0 +1,63 @@
+import numpy
+import openmm
+import openmm.app
+import openmm.unit
+
+from .femto_constants import OpenMMPlatform
+
+def is_close(
+    v1: openmm.unit.Quantity,
+    v2: openmm.unit.Quantity,
+    rtol=1.0e-5,
+    atol=1.0e-8,
+    equal_nan=False,
+) -> bool | numpy.ndarray:
+    """Compares if two unit wrapped values are close using ``numpy.is_close``"""
+
+    if not v1.unit.is_compatible(v2.unit):
+        return False
+    return numpy.isclose(
+        v1.value_in_unit(v1.unit),
+        v2.value_in_unit(v1.unit),
+        atol=atol,
+        rtol=rtol,
+        equal_nan=equal_nan,
+    )
+
+def compute_energy(
+    system: openmm.System,
+    positions: openmm.unit.Quantity,
+    box_vectors: openmm.unit.Quantity | None,
+    context_params: dict[str, float] | None = None,
+    platform: OpenMMPlatform = OpenMMPlatform.REFERENCE,
```

Comment:
This doesn't get used anywhere right now.

Reply:
I would remove the import here and just have folks pass the platforms themselves like in: https://github.com/choderalab/openmmtools/blob/59c14bfc147a4128eebd61f75927875c2b9976ad/openmmtools/tests/test_alchemy.py#L81